cmd/cloud.go

```diff
@@ -231,13 +231,15 @@ This will execute the test on the Load Impact cloud service. Use "k6 login cloud
 		}
 
 		if testProgress == nil {
-			return ExitCode{errors.New("Test progress error"), 98}
+			//nolint:golint
```

Comment:
what is golint saying here ? I would've guessed it saying to you that you can use the names of the fields and skip the last one ... and I agree with that. I would also want this to be the case everywhere as otherwise someone reading this has no idea what `98` is :) 
Also I would argue `98` should be a constant somewhere

Reply:
It said error message should not be capitalized.

Reply:
I would argue that it would've been easier to ... not capitalize the error, also my comment still stands ;). 

For the sake of the argument how does it look capitalized and not capitalized in the terminal ?

Reply:
I think it's not how it look, but golint follows https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
@MStoykov Yes, I'm the same side with out about constant. But it's better to do it in other PR, it's out of context for the purpose of this PR.

(I myself does not know what `98` means)

Reply:
Anyway, if any of you can provide the meaning of the literal code (98, 99 ...) I can make the changes @na-- @MStoykov

Reply:
cc @na--

Reply:
@MStoykov Fixed